.github/workflows/create-releases.yml

```diff
@@ -53,7 +53,7 @@ jobs:
       - name: Set up Go
         uses: actions/setup-go@v2
         with:
-          go-version: 1.15.3
+          go-version: 1.17
```

Comment:
The most current version is go-version: 1.17.6. Does the minor version matter? You used the minor version for 1.15.

Reply:
`1.17` selects automatically the latest 1.17 release - so we have always the latest patches/fixes.
In Github Actions, you can see that currently go1.17.6 is used.